HISTORY.rst

```diff
@@ -36,6 +36,43 @@ unreleased
   are supported. Once registered, the device will show up in the linked IoT Hub and can be interacted with or
   simulated using other `az iot device` commands.
 
+**IoT Central updates**
```

Comment:
Since we published last week, these release notes should go under the `unreleased` section

Reply:
Sure